tgui-next/packages/tgui/components/Collapsable.js

```diff
@@ -0,0 +1,56 @@
+import { Component, Fragment } from "inferno";
+import { classes } from 'common/react';
+import { Box } from './Box';
+import { Icon } from "./Icon";
+
+export class Collapsable extends Component {
```

Comment:
Can you call this "Collapsible", please? 😆

Reply:
I knew it looked wrong